### ISSUE TITLE
Linux installer overwrites menu entries when previous versions installed

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#391]
```
On an Ubuntu (Linux with GNOME desktop) installation, the YAWL4Study installer 
overwrites the Applications ("Start") menu entries for any existing version. 
This occurred with the 2.1beta installer, but fairly sure it's an issue with 
all previous installers as well.

At a GNOME (XDG menus) level, the problem is that the underling XDG files have 
the same generic names with YAWLVERSION in, such as 
user-yawlfoundation-yawl4study_YAWLVERSION.menu. Thus, the new ones overwrite 
the old ones, which had exactly the same names.

From poking around the SVN repo, it looks like you use IzPack and launch4j for 
creating the installer. It also looks like the IzPack files aren't committed to 
SVN for each new release; the tagged 2.1beta release looks to have installer 
files for an old release (might be wrong).

So I suspect that some variable replacement of YAWLVERSION is not getting done 
somewhere: either within IzPack or within some generic Ant script or similar 
that is being used to build the customised-per-release installer files. Due to 
the generic nature of IzPack, I guess the same problem *could* occur in the 
Windows and/or OS X installers (with OS specific manifestations).

The rest of the installation is fine: i.e. it installs in a separate directory, 
and both the old and new versions can be started successfully from their bin 
directory scripts (though not both at the same time, since they always use the 
same ports and URLs; fair enough, but might that be something that it would be 
useful to support in future?? Would certainly be useful for side-by-side 
testing of multiple versions on the same machine).



```

Original issue reported on code.google.com by `monsieur...@gmail.com` on 9 Jul 2010 at 2:40

**Comment by yawlfoundation** [comment]
```
Hi, no we do not use IzPack (bad old days ;)
We'll look into that.

Thanks
```

Original comment by `stephan....@googlemail.com` on 29 Aug 2010 at 1:11

**Comment by yawlfoundation** [comment]
```
This is fixed in the 3.0 installers
```

Original comment by `yawl.mic...@gmail.com` on 11 Sep 2014 at 11:33
* Changed state: **Fixed**
